shopfloor_checkout_package_measurement_mobile/static/src/scenario/checkout_package_measurement.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2022 Camptocamp SA (http://www.camptocamp.com)
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+import {process_registry} from "/shopfloor_mobile_base/static/wms/src/services/process_registry.js";
+
+import {checkout_states} from "/shopfloor_mobile/static/wms/src/scenario/checkout_states.js";
+
+// Get the original template of the checkout scenario
+const checkout_scenario = process_registry.get("checkout");
+const template = checkout_scenario.component.template;
+// And inject the new state template (for this module) into it
+const pos = template.indexOf("</Screen>");
+const new_template =
+    template.substring(0, pos) +
+    `
+
+<div v-if="state_is('package_measurement')">
+
+    <detail-picking
+        :record="state.data.picking"
+        :card_color="utils.colors.color_for('screen_step_done')"
+    />
+    <detail-package
+        :record="state.data.package"
+    />
+
+    <v-form>
+        <v-container>
+            <v-row>
+                <v-text-field
```

Comment:
If I remember properly, v-text-field does not manage properly the display of the virtual keyboard on a mobile device. This should likely be replaced by the searchbar or something alike and you should set the expected type, in this case number, to show the right keyboard

Reply:
The searchbar uses a v-text-field as well. Oh no wait, are you refering to this PR https://github.com/OCA/wms/pull/350/files I see it is not yet merged in 14.0 and the `v-text-field` has been replaced by an `input`element ?

Reply:
@TDu As we have to type something in the field, we don't have to hide the virtual keyboard. So fine to use v-text-field.
Still need to put type number to display right keyboard and prevent to enter chars